src/pykeen/models/baseline/experiment.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
```

Comment:
@cthoyt should this be part of the main pykeen repo? Or should we move it to another repo (within the pykeen org)

Reply:
We can make an independent repo for it just like some of the other side projects. Let's keep it here until everything is done, though! It's nice to have for testing purposes

Reply:
See https://github.com/pykeen/nonparametric-baseline-benchmark